corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -34,27 +34,32 @@ <h2 class="text-hq-nav-header">{% trans "Data Dictionary" %}</h2>
     <div data-bind="saveButton: saveButton"></div>
     <div class="row">
         <div class="col-md-10">
-            <a class="btn btn-info" id="download-dict" href="{% url "export_data_dictionary" domain %}">Export to Excel</a>
-            <a class="btn btn-default" href="{% url "upload_data_dict" domain %}">Import from Excel</a>
+            <a class="btn btn-info" id="download-dict" href="{% url "export_data_dictionary" domain %}">{% trans "Export to Excel" %}</a>
+            <a class="btn btn-default" href="{% url "upload_data_dict" domain %}">{% trans "Import from Excel" %}</a>
             <h3 data-bind="text: $root.activeCaseType()"></h3>
+            <br/>
+            <button data-bind="click: $root.showDeprecated, visible: !showAll()" class="btn btn-info">{% trans "Show Deprecated Properties" %}</button>
```

Comment:
Currently deprecation hides the properties from this page and from the app manager autocomplete and vellum databrowser if they are not used elsewhere in the app. Happy to change the language but not sure what the correct wording is, since it affects other pages hiding seems less great and i think others had been referring to it as deprecation, but im not at all attached to that term. 
cc: @snopoke @czue who had a similar question above